geth/rpc/client_test.go

```diff
@@ -149,98 +151,93 @@ func (s *RPCTestSuite) TestCallRPC() {
 	require := s.Require()
 	require.NotNil(s.NodeManager)
 
-	nodeConfig, err := MakeTestNodeConfig(params.RinkebyNetworkID)
-	require.NoError(err)
+	for _, upstreamEnabled := range []bool{false, true} {
+		s.T().Logf("TestCallRPC with upstream: %t", upstreamEnabled)
```

Comment:
Let's not add logging to tests. I actually want to clean up other tests that printing logs in similar way. Let's tests be tests, no the debug session runners.

Reply:
If run without `-v` and successful it shouldn't print, right?

Reply:
I hope so. But in Ci we still using `-v`, that means more lines to scrolling, and distract attention while looking for the real FAIL messages. Logs made with `Logf` visually similar to test failure messages, so they adds more confusion without any benefit. Anyway, let's deal with it in another PR.